packages/snaps-jest/src/types.ts

```diff
@@ -244,6 +257,16 @@ export type Snap = {
     transaction?: Partial<TransactionOptions>,
   ): Promise<SnapResponse>;
 
+  /**
+   * Send a signature request to the snap.
+   *
+   * @param signature - The signature request object. Contains the params from
+   * the various signature methods, but has an extra `origin` and `signatureMethod` field.
+   * Any missing fields will be filled in with default values.
+   * @returns The response.
+   */
+  sendSignature(signature?: Partial<SignatureOptions>): Promise<SnapResponse>;
```

Comment:
I'm thinking we should rename these to the name of the handler. `sendTransaction` is fine, but `sendSignature` sounds a bit weird 😅. Maybe better if we do:

- request (`onRpcRequest`) - fine as is.
- onTransaction.
- onSignature.
- onCronjob.
- onHomePage.

What do you think?

Reply:
#2118

Reply:
Can you rebase and rename the function?

Reply:
Done